composer.json

```diff
@@ -139,6 +156,10 @@
                 "package-webentry": "2.18.0",
                 "packages": "^0"
             },
+            "docker-executors": {
+                "docker-executor-cdata": "1.0.1",
+                "docker-executor-node-ssr": "1.5.0"
```

Comment:
@caleeli  can you include also docker-executor-java, docker-executor-python and docker-executor-csharp

Reply:
Added the following versions:

```
                "docker-executor-java": "1.0.3",
                "docker-executor-python": "1.0.1",
                "docker-executor-csharp": "1.0.1"
```